### PR TITLE
feat(ui/rate): allow to clear score

### PR DESCRIPTION
**Pull request by chouchouji**
## Checklist

---

- [ ] allow to clear score

### CODE REVIEW

**Review by nonzzz — APPROVED** [review]
LGTM. but short line is better isn't?

**Comment by chouchouji** [comment]
> 

Sorry, I don't quite understand. Is it optimizing the code?

**Comment by nonzzz** [comment]
@chouchouji 

Look like the follow block

```js
// better
if (condit  === xxx) doSometing 

// bad
 
if (condit === xxx) {
   doSometing 
}


```

If your logic is short. so is no need to wrap it in a block.

**Comment by chouchouji** [comment]
> @chouchouji
> 
> Look like the follow block
> 
> ```js
> // better
> if (condit  === xxx) doSomething 
> 
> // bad
>  
> if (condit === xxx) {
>    doSomething 
> }
> ```
> 
> If your logic is short. so is no need to wrap it in a block.

I know it.  Yesterday when I was writing logic, I modified the logic in the if section, but in the end, I implemented it in another way. The original logic in the if section remained unchanged and I forgot to change it back. 
I will modify it later. Thank you!!!

**Comment by chouchouji** [comment]
> @chouchouji
> 
> Look like the follow block
> 
> ```js
> // better
> if (condit  === xxx) doSomething 
> 
> // bad
>  
> if (condit === xxx) {
>    doSomething 
> }
> ```
> 
> If your logic is short. so is no need to wrap it in a block.

done

**Comment by nonzzz** [comment]
@haoziqaq CC

File: packages/varlet-ui/src/rate/Rate.vue [hunk 1]
```diff
@@ -58,6 +58,7 @@ export default defineComponent({
     const { errorMessage, validateWithTrigger: vt, validate: v, resetValidation } = useValidation()
     const { hovering } = useHoverOverlay()
     const currentHoveringValue = ref<number>(-1)
+    const lastScore = ref<number>(Number(props.modelValue))
```
[the remark this answers is not in the document]
Comment (chouchouji) [reply packages/varlet-ui/src/rate/Rate.vue]: done

**Review by BeADre — APPROVED** [review]

**Pull request merged**